mex/extractors/publisher/extract.py

```diff
@@ -2,22 +2,35 @@
 
 from mex.common.backend_api.connector import BackendApiConnector
 from mex.common.logging import logger
-from mex.common.models import AnyMergedModel
+from mex.common.models import MERGED_MODEL_CLASSES_BY_NAME, AnyMergedModel
+from mex.extractors.settings import Settings
 
 
 def get_merged_items() -> Generator[AnyMergedModel, None, None]:
     """Read merged items from backend."""
     connector = BackendApiConnector.get()
+    settings = Settings.get()
 
     response = connector.fetch_merged_items(None, None, None, 0, 1)
     total_item_number = response.total
 
     item_number_limit = 100  # 100 is the maximum possible number per get-request
 
     logging_counter = 0
+
+    allowlist_for_merged_items_to_extract = [
+        item
+        for item in list(MERGED_MODEL_CLASSES_BY_NAME.keys())
```

Comment:
```suggestion
        for item in MERGED_MODEL_CLASSES_BY_NAME
```

iterating over a dict already yields the keys